### DISCUSSION TITLE
eigen version identification bug

### DISCUSSION THREAD

**Discussion by soto-13** [Homebrew/discussions#1345] [Everyday usage] [answered]
Hello

Upon executing `brew cleanup` I get:

`Warning: Skipping eigen: most recent version 3.3.9 not installed`

yet:

```
brew upgrade eigen
Warning: eigen 3.4 already installed
```
I guess brew doesn't find the actual latest version

**Comment by carlocab** [comment, marked as answer]
`eigen` was briefly upgraded to 3.4, but that was reverted because 3.4 was a pre-release. See https://github.com/Homebrew/homebrew-core/issues/75472.

Please do `brew reinstall eigen`. If that doesn't work, try `brew uninstall --ignore-dependencies eigen && brew install eigen`. You'll need to do the same with `ceres-solver`, if you have it installed.

Apologies for the inconvenience!

**Reply by soto-13** [reply]
no problem, thanks for the prompt response
`brew reinstall eigen` worked fine and the issue is fixed
thanks!
